.eslintrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
         SwitchCase: 1,
       },
     ],
+    'lit-a11y/role-has-required-aria-attrs': 0,
```

Comment:
I disabled this because of this code here:

https://github.com/GoogleChrome/web.dev/blob/a40cb7161d4bf00147fefc63796b9c694b57c2a3/src/lib/components/Search/index.js#L99

According to https://www.w3.org/TR/wai-aria-1.1/#combobox , an element with `role="combobox"` requires `aria-controls`, but it seems like using `aria-owns` is valid as well, so I'm not entirely sure how to go about this, both in this project, as well as in the linting plugin... Maybe @robdodson could help shine a light?

Reply:
it seems like you can enable this rule and our comboxbox should set aria-controls on `.web-search__input`

Reply:
ah actually we'll have to give web-search__input an id because aria only works with id references.